lib/CL/clCreateUserEvent.c

```diff
@@ -1,15 +1,18 @@
 #include "pocl_cl.h"
 #include "pocl_util.h"
-
+#include "pocl_timing.h"
 
 CL_API_ENTRY cl_event CL_API_CALL
 POname(clCreateUserEvent)(cl_context     context ,
                   cl_int *       errcode_ret ) CL_API_SUFFIX__VERSION_1_1
 {
-  int error;
+  int error; 
   cl_event event = NULL;
 
-  error = pocl_create_event (&event, context, NULL, CL_COMMAND_USER);
+  error = pocl_create_event (&event, 0, CL_COMMAND_USER, 0, NULL, context);
+  
+  event->context = context;
+  event->status = CL_QUEUED;
```

Comment:
At this point event is being created and CL_QUEUED is just initial value. Didn't want to initialize to an undefined value if that value became defined in some point in future. POCL_UPDATE_EVENT_QUEUED is called later when associated command is actually queued.